### DISCUSSION TITLE
VExtensions Discussions

### DISCUSSION THREAD

**Discussion by vurvdev** [vurvdev/VExtensions#22] [General]
## Discussions thing
Testing out this discussions thing. 
Seems really neat!
Thanks for letting me know about it @Cheatoid, was actually thinking about GitHub having a feature so there wasn't a need for useless PR's / Issues if someone had a question

Ask, do, say whatever you want here, as long as it is about VExtensions and complies with the TOS

**Comment by vurvdev** [comment]
Just looks like a forum lol

**Reply by Cheatoid** [reply]
Yeah indeed 😄
